scaleway/data_source_account_project_test.go

```diff
@@ -0,0 +1,91 @@
+package scaleway
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccScalewayDataSourceAccountProject_Basic(t *testing.T) {
+	SkipBetaTest(t)
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+	orgID, orgIDExists := tt.Meta.scwClient.GetDefaultOrganizationID()
+	if !orgIDExists {
+		orgID = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+	}
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		CheckDestroy: resource.ComposeTestCheckFunc(
+			testAccCheckScalewayAccountProjectDestroy(tt),
+		),
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+					resource scaleway_account_project "project" {
+						name = "test-terraform-account-project"
+					}
+
+					data scaleway_account_project "by_name" {
+						name = scaleway_account_project.project.name
+						organization_id = "%s"
```

Comment:
Organization id is required for this request, normally you can skip this field as it will fetch with your default orgID, but acceptance tests are run without orgID. I mocked organization ID in the test if there is not default.